games-emulation/pcsx2/pcsx2-1.3.1.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+
+inherit cmake-utils l10n multilib wxwidgets
+
+KEYWORDS="-* ~amd64 ~x86"
+SRC_URI="https://github.com/PCSX2/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+DESCRIPTION="A PlayStation 2 emulator"
+HOMEPAGE="http://www.pcsx2.net"
+
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE="+wxwidgets3"
```

Comment:
Do we need that much of micro-management? For gtk2 vs gtk3 conversion the majority of the dev community was pro "pick the best supported version for the particular package and avoid exposing gtk2/gtk3 USE flags to the user".